src/app/app.component.html

```diff
@@ -36,7 +36,7 @@
   </div>
   <div class="hidden md:flex items-center">
     <div>
-      <img src="assets/playschool-top-icon-email@2x.png">
+      <img src="assets/playschool-top-icon-email@2x.png" alt="">
```

Comment:
Mijn voorstel is om de alt alsnog te vullen met e-mail icon. Een lege alt tekst voegt niet veel toe.

Reply:
Het idee is dat de `<b>Email</b>` 3 regels verder opgelezen wordt door screenreaders, gevolgd door het daadwerkelijke e-mail adres. Als we het icon nog op laten lezen dan wordt de tekst "Email icon Email info@littil.org".

Dus omdat het icon ook daadwerkelijk niks toevoegd hier behalve op grafisch vlak hebben we hier de alt-tekst leeg gelaten.